packages/fxa-settings/src/components/Ready/index.tsx

```diff
@@ -48,9 +46,8 @@ const Ready = ({
               type="submit"
               className="cta-primary cta-base-p font-bold mx-2 flex-1"
               onClick={(e) => {
```

Comment:
The `h1` in this file should have a class of `card-header` instead of `text-xl mb-2`.